x-pack/plugins/session_view/public/components/session_view/index.tsx

```diff
@@ -37,6 +40,10 @@ export const SessionView = ({ sessionEntityId, height, jumpToEvent }: SessionVie
 
   const [searchQuery, setSearchQuery] = useState('');
   const [searchResults, setSearchResults] = useState<Process[] | null>(null);
+  const [optionsStates, setOptionsStates] = useState<StateField>({
```

Comment:
did we maybe want to call this [displayOptions, setDisplayOptions] ? I feel including the word States is a bit "pardon the pun" verbose. Same for StateField. The name doesn't really describe what it is. DisplayOption might be better?